extension/background/background.js

```
@@ -0,0 +1,130 @@
+      return true; // Keep the message channel open for async response
+      
+    case 'sendChatRequest':
+      // Forward the chat request to the API
+      handleChatRequest(message.data)
+        .then(response => sendResponse(response))
+        .catch(error => sendResponse({ error: error.message }));
+      return true; // Keep the message channel open for async response
+      
+    case 'validateApiKey':
+      // Validate the API key format
+      const isValid = validateApiKey(message.apiKey);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)